PowerUp/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="black">#000000</color>
```

Comment:
Black color is already defined in android so you can reference it as `@android:color/black` and remove this